cwrc_audit_report.rb

```diff
@@ -93,9 +93,9 @@ module CWRCPerserver
 
   # Iterate via markers
   # https://github.com/ruby-openstack/ruby-openstack/blob/d9c8aa19488062e483771a9168d24f2626fe688b/lib/openstack/swift/container.rb#L100
-  swift_objs = swift_container.objects_detail
+  swift_objs = swift_container.objects
   while swift_objs.count < swift_container.container_metadata[:count].to_i
-    swift_objs = swift_objs.merge(swift_container.objects_detail(marker: swift_objs.keys.last))
+    swift_objs = swift_objs.merge(swift_container.objects(marker: swift_objs.keys.last))
   end
```

Comment:
refactor: `last-mod-timestamp` metadata not returned by `objects_detail` call therefor refactor to use the more succinct `objects` call returning a array as opposed to a hash.